Execution/CMakeLists.txt

```diff
@@ -1,16 +1,25 @@
 ## Execution CMakeLists.txt
 
 set(LIBRARY_NAME execution)
-set(FLAGS_CROSS "-D__cdecl=\"\" -D__stdcall=\"\"")
+if(WIN32)
```

Comment:
Wrong if statement. Same condition for if and else

Reply:
I find it weird that calling conventions get smashed on linux. Are you sure this is your intention?

Reply:
It should work with `__attribute__((stdcall))` and `__attribute__((cdecl))`. But I see the definition is for `_cdecl` and `_stdcall`. The previous code uses `__stdcall` and `__cdecl`. Check for example `revtracer/revtracer.h:70`